src/gameplay/game.ts

```diff
@@ -1323,16 +1342,31 @@ class Game extends Room {
       }
     });
 
-    // store player data:
-    const timeFinished = new Date();
-    const timeStarted = new Date(this.startGameTime);
-
-    const gameDuration = new Date(timeFinished - timeStarted);
+    //FR2 - Rating system 1 - record games for rating updates   
+    // This data will be used to calculate ratings at end of rating period.
+    // The intended workflow is games -> ratingPeriodGameRecord -> (at end of rating preiod) Ratings Calculator
+
+    if (this.roomCreationType === roomCreationTypeEnum.RANKED_QUEUE) {
+      const ratingRecordToStore = {
+        timeGameStarted: this.startGameTime,
+        timeGameFinished: timeFinished,
+        winningTeam: this.winner,
+        spyTeam: this.spyUsernames,
+        resistanceTeam: this.resistanceUsernames,
+        numberOfPlayers: this.playersInGame.length,
```

Comment:
We can add an assert above and rm this line.

Reply:
Would this be okay?
```
    assert.equal(this.spyUsernames.length + this.resistanceUsernames.length, this.playersInGame.length, 
      'Number of players should be number of resistance players + number of spy players'
    );
```